images/router/haproxy/conf/haproxy-config.template

```diff
@@ -239,7 +239,11 @@ backend be_tcp_{{$cfgIdx}}
 {{ if ne (env "ROUTER_SYSLOG_ADDRESS" "") ""}}
   option tcplog
 {{ end }}
+{{ if eq (env "ROUTER_DISABLE_STICKY_SESSIONS" "") "" }}
```

Comment:
Didn't want to use a user-supplied value directly as that could potentially break config. Or did you mean something else?